src-tauri/src/app_config.rs

```diff
@@ -232,7 +232,6 @@ pub(crate) struct AppConfig {
     created_at: Option<DateTime<Utc>>,
     mode: MiningMode,
     display_mode: DisplayMode,
-    auto_mining: bool,
```

Comment:
I think this will break existing configs. The value in existing configs won't be ignored, it'll consider the config invalid and fail to parse it, and reset the entire config to defaults. 

This needs to be validated before merge.

Reply:
I think I'm wrong. It will only break if it was a value in `AppConfigFromFile`